notation/errors.go

```diff
@@ -0,0 +1,49 @@
+package notation
+
+// ErrorVerificationInconclusive is used when signature verification fails due to a runtime error (e.g. a network error)
+type ErrorVerificationInconclusive struct {
```

Comment:
It's here because both notation package and verifier package use it. verifier package also imports notation package. If move errors.go to verifier package, there is import cycle.

Reply:
Why does verify needs to depend on notation? can we break that circular dependency?

Reply:
That's because `verifier.New()` returns a `notation.Verifier` by design.